packages/component-elife-app/index.js

```diff
@@ -15,6 +16,9 @@ const stopServer = () => {
 }
 
 logger.info(appMessage('Starting'))
+logger.info(
+  `Feature status: ${JSON.stringify(config.get('features'), null, 4)}`,
```

Comment:
Would it be better to say `Features enabled:` as we are not showing its current status?

Reply:
... so, great minds think alike - I started with this ... but then it sort of didn't look right with :
```
info: Features enabled: {
    "demographicSurvey": false
}
```
So you would prefer this than:
```
info: Feature status: {
    "demographicSurvey": false
}
```
?